### PR TITLE
Make VersionInfo [Experimental] until it is fully working

### PR DESCRIPTION
**Pull request by bpulliam**
This change returns the VersionInfo API to experimental, exactly the way we shipped 1.2-stable.
It should stay this way until the API is fully implemented, tested, and verified.

A microsoft employee must use /azp run to validate using the pipelines below.

WARNING:
Comments made by azure-pipelines bot maybe inaccurate. 
Please see pipeline link to verify that the build is being ran.

For status checks on the develop branch, please use TransportPackage-Foundation-PR
(https://microsoft.visualstudio.com/ProjectReunion/_build?definitionId=81063&_a=summary)
and run the build against your PR branch with the default parameters.

For status checks on the main branch, please use microsoft.ProjectReunion
(https://dev.azure.com/ms/ProjectReunion/_build?definitionId=391&_a=summary)
and run the build against your PR branch with the default parameters.

### CODE REVIEW

File: test/VersionInfo/VersionInfoTests.cpp [hunk 1]
```diff
@@ -30,6 +30,12 @@ namespace Test::VersionInfo
 
         TEST_METHOD(VersionInfo_Release)
         {
+            if (::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::Feature_VersionInfoAPI::IsEnabled())
+            {
+                WEX::Logging::Log::Comment(WEX::Common::String(L"Feature_VersionInfoAPI is disabled. Skipping..."));
+                return;
+            }
```
Comment (DrusTheAxe) [inline test/VersionInfo/VersionInfoTests.cpp:37]: Better in the SETUP fixture, then it skips all tests with the 1 check

**Review by DrusTheAxe — APPROVED** [review]
Change the tests to set Skipped in the SETUP fixture (see comment) but otherwise LGTM

**Review by codendone — APPROVED** [review]
:shipit:

**Pull request merged**